socorro/siglists/irrelevant_signature_re.txt

```diff
@@ -21,7 +23,12 @@ data@app@org\.mozilla\.f.*-\d\.apk@classes\.dex@0x.*
 framework\.odex@0x.*
 GetLCIDFromLangListNodeWithLICCheck
 gfxPlatform::GetPlatform
-google_breakpad::ExceptionHandler::HandleInvalidParameter.*
+glFlush_ExecThread
+glDrawArrays.*
```

Comment:
My reading of @luser's [last comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1410580#c19) is that we want both the `gl` lines above in signatures, wouldn't including them in the irrelevant list remove them?

Reply:
Yeah. So, some of the comments in that bug suggest this (and the other `gl*` ones) should be in. And some suggest they should be out. And honesty, I feel like we're playing the hokey-pokey at this point.

I'm not qualified to state with confidence what should be in/out. However, seems better to hedge towards not including things in the irrelevant list that we're not confident should definitely be there.

I pinged @luser to sanity-check this one more time. I'm loathe to do more iterations on change, then comment in the bug, then get a consensus since this spohl indicated this was critical and should get fixed asap. I'm thinking we'll want to reprocess the last week's worth of crashes, too.

That's about where I'm at.